test-runner.js

```diff
@@ -1,5 +1,9 @@
+require('babel-polyfill');
+
 const testsContext = require.context('./tests', true, /\.js$/);
 const componentsContext = require.context('./src/', true, /components\/.*\.js$/);
+const reducersContext = require.context('./src/', true, /reducers\/.*\.js$/);
```

Comment:
Thinking about this and after seen the containers directory - it might be easier to expand the regex and have one thing to deal with all the bits or just exclude what we can't run here (e.g. the server bits) that would save us needing to update it all the time.